lib/cog/repository/bundles.ex

```diff
@@ -17,9 +17,14 @@ defmodule Cog.Repository.Bundles do
   @reserved_bundle_names [
     Cog.embedded_bundle,
     Cog.site_namespace,
-    "user" # a bundle named "user" would break alias resolution
+    "user", # a bundle named "user" would break alias resolution
+    "cog"   # we're going to squat on this for now to prevent potential confusion
   ]
 
+  # We're also going to block the installation of any bundles that are
+  # prefixed with "cog" or the embedded bundle name
+  @reserved_bundle_name_regex Regex.compile!("^(cog|#{Cog.embedded_bundle})-.*$")
```

Comment:
I think handling the names "cog" and "operable" are good enough for now. We can change this in a future release if needed.